tests/functional/events/16-task-event-job-logs-register-globalcfg/flow.cylc

```diff
@@ -8,14 +8,13 @@
 
 [runtime]
     [[t1]]
+        init-script = cylc__job__disable_fail_signals ERR EXIT
         script="""
-wait "${CYLC_TASK_MESSAGE_STARTED_PID}" 2>/dev/null || true
+cylc__job__wait_cylc_message_started
 mv "$0.out" "$0.out.keep"
 mv "$0.err" "$0.err.keep"
 cylc message 'succeeded' >>"$0.out.keep"
 sleep 5
 cp -p "$0.out.keep" "$0.out"
 cp -p "$0.err.keep" "$0.err"
-
-trap '' EXIT
```

Comment:
This suite doesn't have any associated test case, so can be removed.

Reply:
Good spotting :+1: